jfather.json.txt

```diff
@@ -0,0 +1,116 @@
+{
```

Comment:
1. This is a JSON file but it's saved as a `.txt` file
2. Don't commit your data, add it to `.gitignore`